modules/ding_permissions/ding_permissions.module

```diff
@@ -1011,3 +1013,14 @@ function ding_permissions_secure_permissions($role) {
     return $permissions[$role];
   }
 }
+
+/**
+ * Implements hook_page_alter().
```

Comment:
Would it be not be possible to just use one of the toolbars? Simply disabling the one you don't want to use?